cmd/stack_validate.go

```diff
@@ -43,15 +43,16 @@ func newStackValidateCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var stackValidateCmd = &cobra.Command{
 		Use:   "validate",
-		Short: "Run validation tests against a stack in the local Appsody environment",
-		Long: `This command is a tool for stack developers to validate a stack from their local Appsody development environment. It performs the following against the stack:
-- Runs the stack lint test. This can be turned off with the --no-lint flag
-- Runs the stack package command. This can be turned off with the --no-package flag
-- Runs the appsody init command
-- Runs the appsody run command
-- Runs the appsody test command
-- Runs the appsody build command
-- Provides a Passed/Failed status and summary of the above operations`,
+		Short: "Run validation tests against your stack.",
+		Long: `A tool for stack developers to validate your stack in their local Appsody development environment. 
```

Comment:
@doveye A stack is something "stack creators" create (we somewhat maintain) and "stack creators" will validate

Reply:
When users run `appsody init <stack>`, they will get a project template specified in the stack rather than the stack itself, so these `appsody stack` commands are mostly used by "stack creators/maintainers"

Reply:
Updated as suggested.